docs/samples/registries.conf

```diff
@@ -0,0 +1,25 @@
+# This is a sample of system-wide configuration file used to
+# keep track of registries for various container backends.
+# It adheres to TOML format and does not support recursive
+# lists of registries.
+
+# The default location for this configuration file is /etc/containers/registries.conf.
+
+# The only valid categories are: 'registries.search', 'registries.insecure',
+# and 'registries.block'.
+
+[registries.search]
+registries = ['docker.io', 'quay.io']
```

Comment:
I'd flip the order, put quay first, docker second

Reply:
fixed :raising_hand_woman: